### PR TITLE
feat: add vue

### PR DESCRIPTION
**Pull request by digitalsadhu**
Adds a new folder for Vue. 
Uploads all files in the Vue dist folder to the Eik server. Starting with Vue 2, once in and published, we should get updates to update to v3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "license": "ISC",
   "workspaces": [
     "packages/react",
-    "packages/react-dom"
+    "packages/react-dom",
+    "packages/vue"
```
Comment (trygve-lie) [inline package.json:16]: You can actually do `packages/*` instead of explicitly defining every package.

Comment (digitalsadhu) [reply package.json:16]: When you run `npm init -w packages/vue` it automatically creates this entry in the root `package.json`. Thinking that if we define it as a glob then that's gonna get messed up a bit as the init command will still add a new entry for each new folder.

Comment (trygve-lie) [reply package.json:16]: Ack :+1:

**Review by trygve-lie — APPROVED** [review]
lgtm :100:

**Pull request merged**